packages/javascript-decorators/README.md

```diff
@@ -0,0 +1,117 @@
+# `@shopify/javascript-decorators`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fjavascript-decorators.svg)](https://badge.fury.io/js/%40shopify%2Fjavascript-decorators.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/javascript-decorators.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/javascript-decorators.svg)
+
+A set of decorators to aid your javascript journey.
+
+## Installation
+
+```bash
+$ yarn add @shopify/javascript-decorators
```

Comment:
Nit: these can be used with TypeScript!   Should it be `@shopify/decorators`?

Reply:
was trying to be more specific, my assumption being that npm != javascript?